primitives/src/bonding/ledger.rs

```diff
@@ -29,9 +29,9 @@ use frame_support::pallet_prelude::*;
 #[derive(PartialEq, Eq, Clone, Encode, Decode, RuntimeDebug, MaxEncodedLen, TypeInfo)]
 pub struct UnlockChunk<Moment> {
 	/// Amount of funds to be unlocked.
-	value: Balance,
+	pub value: Balance,
```

Comment:
instead of making the fields pub, we can have a pub method to return the values as a tuple or ethabi encoded value